hazelcast/src/main/java/com/hazelcast/sql/impl/client/SqlError.java

```diff
@@ -26,11 +27,24 @@
     private final int code;
     private final String message;
     private final UUID originatingMemberId;
+    private final boolean suggestionExists;
+    private final String suggestion;
 
     public SqlError(int code, String message, UUID originatingMemberId) {
+        this(code, message, originatingMemberId, false, null);
+    }
+
+    public SqlError(
+            int code,
+            String message,
+            UUID originatingMemberId,
+            boolean suggestionExists, String suggestion
```

Comment:
Just curious, but is there any reason (aside from style/simplifying `equals` checks) not to just use a `suggestion != null` instead of `suggestionExists`?

Reply:
We have to support `v1.equals(v2)` as well as `v2.eauals(v1)`.